interfaces/systemd/backend.go

```diff
@@ -78,17 +78,21 @@ func (b *Backend) Setup(snapInfo *snap.Info, devMode bool, repo *interfaces.Repo
 			logger.Noticef("cannot reload systemd state: %s", err)
 		}
 	}
-	// Start any new services
+	// Start and enable any new services
 	for _, service := range changed {
-		err := systemd.Start(service)
-		if err != nil {
+		if err := systemd.Enable(service); err != nil {
```

Comment:
Per our conversation online, this looks good, and I wasn't even aware it was necessary. I naively assumed `systemctl daemon-reload` would create the symlinks automatically, as long as the service hadn't been explicitly disabled, but clearly not reading the docs.

As a minor improvement on top of this, can we please introduce `systemd.EnableNow/DisableNow`, which simply include `--now` in the command line, so we have half as many systemctl calls?